code/sorting/src/cocktail_sort/cocktail_sort.c

```diff
@@ -0,0 +1,67 @@
+#include "stdio.h"
+#include "conio.h"
+
+/* CocktailSort function is the function used to sort the array using cocktail algorithm. */
+/* Cocktailsort is similar to the bubble sort but the complexity is less comparatively. */
+void CocktailSort(int l[], int n)  
+{   
+    /* argument passed is the list of the array to be sorted and the length of the array */
+    /* first is used to represent the starting index of the array
+     * last is used to represent the last index of the array
+     * p is the variable used to check if the array contains sorted elements after sorting process
+     */
+    int first = 0;
+    int last = n-1;
+    int temp = 0;
```

Comment:
Reformat this file using clang format 4 spaces.

Reply:
@arnavb  sir,would you please elaborate what exactly is to be done?

Reply:
sir, now it's indented to 4 spaces. please look at it. @arnavb